.github/workflows/build.yml

```diff
@@ -33,4 +33,6 @@ jobs:
     - name: install dependencies
       run: sudo apt-get --quiet install ninja-build devscripts
     - name: build.py
-      run: ./src/build.py --sync-include=cmake,wabt --build-include=wabt,debian --no-test --no-host-clang --no-sysroot
+      run: ./src/build.py --sync-include=sysroot,cmake,wabt --build-include=wabt,debian --no-test --no-host-clang --no-sysroot
```

Comment:
do we want sysroot in the sync if it builds with --no-sysroot?

Reply:
Ooops.